src/main/java/com/sobytylnik/Profile.java

```diff
@@ -0,0 +1,76 @@
+package com.sobytylnik;
+
+import java.util.Objects;
+import java.util.Random;
+
+public class Profile {
+    private Long id;
+    private String name;
+    private String surname;
+    private int age;
+
+    public Profile(String name, String surname, int age) {
+        this.id = id;
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public void setName(String newName) {
+        name = newName;
+    }
+
+    public void setSurName(String newSurName) {
+        surname = newSurName;
+
+    }
```

Comment:
Same as for `setName()`: follow classic setter convention.
Surname is single word (not sur name).